examples/6-concurrent-requests.php

```diff
@@ -16,7 +16,7 @@
     ];
 
     // Instantiate the HTTP client
-    $client = new Client;
+    $client = HttpClientBuilder::ofPool()->build();
```

Comment:
Maybe we should add another static method `create()` / `createDefault()` / `buildDefault()` which is a shortcut for `::ofPool()->build()`?

Reply:
I think so, yes. +1 for `buildDefault()`.